C/c4DocEnumerator.cc

```diff
@@ -31,7 +31,7 @@ using namespace litecore;
 #pragma mark - DOC ENUMERATION:
 
 CBL_CORE_API const C4EnumeratorOptions kC4DefaultEnumeratorOptions = {
-    kC4IncludeNonConflicted | kC4IncludeBodies
+        kC4IncludeNonConflicted | kC4IncludeBodies
```

Comment:
Why is this indented two levels at once?

Reply:
Fixed with move to Clang-Format